src/drivers/i2c/adapters/stm32/i2c_conf_f4.h

```diff
@@ -21,7 +21,7 @@
 #include <module/embox/driver/i2c/stm32_i2c_f4.h>
 
 /* I2C1 */
-#define I2C1_SCL_PIN  GPIO_PIN_6
+#define I2C1_SCL_PIN  GPIO_PIN_8
```

Comment:
It is rather dangerous changes at this moment. Please exclude this commit from the PR

Reply:
Ok